oap-stdlib/src/main/java/oap/application/KernelHelper.java

```diff
@@ -171,4 +176,12 @@ public static void restoreThreadName() {
         }
 
     }
+
+    record Parameter(Object value, boolean ignoreCast) {
+    }
+
+    static class Parameters {
+        public final LinkedHashMap<String, Object> configurationParameters = new LinkedHashMap<>();
+        public final LinkedHashMap<String, Object> serviceParameters = new LinkedHashMap<>();
```

Comment:
What is the difference between `configurationParameters` and `serviceParameters`?